packages/compiler/src/expression_parser/parser.ts

```diff
@@ -150,7 +169,7 @@ export class Parser {
     const tokens = this._lexer.tokenize(templateValue);
     const parser = new _ParseAST(
         templateValue, templateUrl, absoluteValueOffset, tokens, templateValue.length,
-        false /* parseAction */, this.errors, 0 /* relative offset */);
+        ParseFlags.None /* parseFlags */, this.errors, 0 /* relative offset */);
```

Comment:
I think the comment is unnecessary now
```suggestion
        ParseFlags.None, this.errors, 0 /* relative offset */);
```